web-registry/src/pages/Home/Home.tsx

```diff
@@ -57,13 +58,12 @@ const Home: React.FC = () => {
   return (
     <Box sx={{ backgroundColor: 'primary.main' }}>
       <SEO
-        title="Regen Marketplace"
-        description=""
-        // TODO: add link to preview image
-        imageUrl="TODO"
+        title={seo?.title || ''}
+        description={seo?.description || ''}
+        imageUrl={seo?.image?.asset?.url || ''} // TODO: specify dimensions here. See: https://www.sanity.io/docs/image-urls
```

Comment:
@wgwz note the comment here - we might want to specify dimensions for the image, which is as easy as adding some query params to the end of this string ie something like
```suggestion
        imageUrl={seo?.image?.asset?.url + ‘?&h=800&w=1600’}
```
(that’s pseudocode - probably a bit off)